SpacemanDMM.toml

```diff
@@ -51,3 +51,22 @@ macro_undefined_no_definition = "error"
 
 # Raised by Object Tree
 override_precedes_definition = "error"
+
+[map_renderer]
+hide_invisible = [
+	"/obj/effect/step_trigger",
+	"/obj/abstract",
+]
+
+[map_renderer.render_passes]
+overlays = false
+gravity-gen = false
+pretty = false
+icon-smoothing = false
+icon-smoothing-2016 = true
+smart-cables = false
+# TODO: add support for directional_offsets var to Jupyterkat's fork?
+offsets = false
+# maybe also add support for our random spawners?
```

Comment:
Hmm. For TG random spawners it replaces things like random vendor spawners with a vendor, etc. I'm not sure how feasible that is. It could be useful to enable for things like regenerating the nanoui maps, though.

I also need to add a TODO about a Bay-specific wall smoothing pass